### PR TITLE
55528 - Implement Login Authorization & Password Change Functionality

### PR DESCRIPTION
**Pull request by RobertNugent04**
In this branch, I have implemented the APIs for the following functionalities:

**1. Authorizing Login:**

- Engineers can use their credentials to authenticate against the system.
- The system validates these credentials and returns a JWT token upon successful authentication.

**2. Changing an Admin's Password:**

- Use of an API endpoint has been implemented to allow admins to change their password.

**3. Creating Token for Engineer**

- Allowing an Admin to create a token for an engineer to login with through use of an API

Scenario Images:

Creating an engineer token:
create token

Logging In:
logging

Changing Password:
changing password

### CODE REVIEW

**Comment by gursimar03** [comment]
@RobertNugent04  Please add some screenshots of different scenarios. 
Have you also created a page for creating engineer token as well?

File: upflux-client/src/api/LoginRequests.ts
Comment (gursimar03) [inline upflux-client/src/api/LoginRequests.ts:1]: please rename this file as lowercase... use git command for renaming files and sometimes using UI may bug the git out.

File: upflux-client/src/api/apiConsts.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const API_BASE_URL = 'https://localhost:7034/api';
```
Comment (gursimar03) [inline upflux-client/src/api/apiConsts.ts]: As we discussed regarding the API_BASE_URI. Could you please update the code so it changes based on the environment it is ran? Thanks

**Review by Erlinguss — DISMISSED** [review]
Looks great! UI on fire

**Review by gursimar03 — DISMISSED** [review]

**Review by gursimar03 — DISMISSED** [review]

**Review by gursimar03 — APPROVED** [review]

**Review by Erlinguss — APPROVED** [review]
Code reviewed.

**Pull request merged**